src/utils/options.js

```diff
@@ -35,10 +35,11 @@ export function number(val: mixed, name: string): number | void {
 export function toSpawnOpts(flags: Flags): SpawnOpts {
   let spawnOpts = {};
 
-  if (flags.parallel && flags.serial) {
+  if ((flags.parallelNodes || flags.parallel) && flags.serial) {
```

Comment:
Probably no need to check for parallel **and** parallelNodes I guess?

Reply:
Parallel nodes kinda implies parallel so I'm not really worried about them both being specified